website/common/script/ops/scoreTask.js

```diff
@@ -189,6 +189,37 @@ function _updateCounter (task, direction, times) {
   }
 }
 
+function _lastHistoryEntryWasToday (lastHistoryEntry, user) {
```

Comment:
are the changes in this file expected?

Reply:
No, this commit is not related to the issue, it's just something I stumbled upon. There is no change in logic, just (I hope) improvement in readability. But I can discard this commit if you want or make a separate branch for refactoring

Reply:
ah okay, let's leave it here since it's definitely and improvement!